### ISSUE TITLE
Trim lines for terminal length and newlines.

### ISSUE THREAD

**Issue by glennpratt** [askreet/interactive-logger#1]
Lines that wrap on the terminal or have newlines cause the output to repeat instead of being replaced.

Here is how I trim for length, but watching @jamesiarmes use this on a huge terminal, I noticed we also should trim at newlines.

``` ruby
    if msg.length > (terminal_width - 18)
      msg = "#{msg[0..(terminal_width - 22)]}..."
    end
```

**Comment by glennpratt** [comment]
In that case, terminal_width is from Thor, we can also get it from Rake.

https://github.com/erikhuda/thor/blob/master/lib/thor/shell/basic.rb#L271-L282

Might make sense to try and guess if one or the other is active (rake or thor).

**Comment by askreet** [comment]
Fixed in #5
